Examples/nl-core-patient-lifeStance - example.xml

```diff
@@ -0,0 +1,158 @@
+<?xml-model href="http://hl7.org/fhir/STU3/patient.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
```

Comment:
This file should be renamed, probably to 'example-nl-core-patient-lifeStance-01.xml'. @ArdonToonstra ?

Reply:
You're right. I'll do that.